berkeley-mobile/Common/Images/ImageTableViewCell.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ImageViewCell.swift
```

Comment:
Small nit, but this file is named `ImageTableViewCell.swift`, which I'm assuming is the old name.